src/pages/OrganizationProjectsPage/OrganizationProjectsPage.tsx

```diff
@@ -48,6 +48,9 @@ const SHOULD_INCLUDE_DEPRECATED = true;
 type TOrganizationOptions = {
   orgLabel: string;
 };
+type TProjectResponseCommonExtended = ProjectResponseCommon & {
```

Comment:
ERROR in /home/dinika/code/work/nexus-web/src/pages/OrganizationProjectsPage/OrganizationProjectsPage.tsx
[tsl] ERROR in /home/dinika/code/work/nexus-web/src/pages/OrganizationProjectsPage/OrganizationProjectsPage.tsx(289,9)
      TS2322: Type 'ProjectResponseCommon[]' is not assignable to type 'TProjectResponseCommonExtended[]'.
  Type 'ProjectResponseCommon' is not assignable to type 'TProjectResponseCommonExtended'.
    Property '_markedForDeletion' is missing in type 'ProjectResponseCommon' but required in type '{ _markedForDeletion: boolean; }'